pytest.ini

```diff
@@ -10,3 +10,4 @@
 pep8ignore = docs/conf.py ALL
 addopts = --pep8 --doctest-glob="*.rst" --doctest-modules --cov=invenio_app_rdm --cov-report=term-missing --ignore=setup.py
 testpaths = docs tests invenio_app_rdm
+filterwarnings = ignore::pytest.PytestDeprecationWarning
```

Comment:
Just out of clarity, in case someone else is wondering... This is pytest 6 fault: https://github.com/inveniosoftware/pytest-invenio/issues/54

Release 3 days ago, such a **classic** :joy:

Invenio wide should be addressed in the next sprint.